website/docs/progressive-delivery/progressive-delivery-flagger-install.mdx

```diff
@@ -9,88 +9,75 @@ import TierLabel from "../_components/TierLabel";
+![blue green mirroring icon](/img/mirroring.svg)  **Blue/Green with Traffic Mirroring**: sends copies of incoming requests to the new version of an application. The user receives the response from the current application and the other is discarded. The new version is promoted only if metrics are healthy.
+
+This guide uses Flux manifests to install Flagger and [Linkerd](https://github.com/linkerd), a [CNCF](https://www.cncf.io/) project and service mesh for Kubernetes and beyond. We will walk you through a full end-to-end scenario where you will:
 - [Install the Linkerd service mesh](#installing-linkerd-using-flux)
 - [Install Flagger](#installing-flagger-using-flux)
 - [Deploy a sample application using a canary release strategy based on metrics provided through
   Linkerd's in-built Prometheus instance](#deploy-a-canary-release)
 
 ## Prerequisites
-- This guide assumes you already have a Kubernetes cluster running and have bootstrapped Flux. To
-  apply the manifests listed in this guide, you will need to commit them to a repository being
-  reconciled with Flux. For help installing Flux, you can follow their [getting
-  started](https://fluxcd.io/docs/get-started/) documentation.
-- Flagger requires the `autoscaling/v2` or `autoscaling/v2beta2` API to be installed on the cluster, you can use `kubectl
-  api-resources` to check which API versions are supported.
-- The [step](https://smallstep.com/cli/) CLI installed to generate certificates in order to support
-  mTLS connections.
-
-## Installing Linkerd using Flux
-
-For the Linkerd installation, a Kustomization file will be used. This will allow us to specify the
-installation order and the default namespace for the installed resources but also to easily generate
-Secrets from certificate files via the use of a `secretGenerator`.
-
-In order to support mTLS connections between meshed pods, Linkerd requires a trust anchor
+- This guide assumes you already have a Kubernetes cluster running and have [bootstrapped Flux](https://fluxcd.io/docs/get-started/). To apply the manifests listed here, you will need to commit them to a repository being reconciled with Flux.
+- Flagger requires the [`autoscaling/v2`](https://github.com/kubernetes/api/tree/master/autoscaling/v2) or [`autoscaling/v2beta2`](https://github.com/kubernetes/api/tree/master/autoscaling/v2beta2) API to be installed on your cluster. You can use `kubectl api-resources` to check which API versions are supported.
+- The [step](https://smallstep.com/cli/) CLI installed to generate certificates that support mTLS connections.
+
+## Installing Linkerd Using Flux
+
+To install Linkerd we'll use a Kustomization file. It will allow us to specify the order and default namespace for the installed resources, and to easily generate Secrets from certificate files via the use of a `secretGenerator`.
```

Comment:
```suggestion
To install Linkerd we'll use a Kustomization file. It will allow us to specify the order and default namespace for the installed resources, and to generate Secrets from certificate files via the use of a `secretGenerator`.
```

(not up to us to tell the user what's easy and what's not)

Reply:
Agree, good one